### ISSUE TITLE
Migrate from JCenter artifacts

### ISSUE THREAD

**Issue by vbedrosova** [rodm/gradle-teamcity-plugin#68]
As described in [this article](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/) JCenter is being gradually shut down. Due to a possible blackout we've been having issues building our TeamCity plugins recently with errors like:

```
       > Could not download plugin-repository-rest-client-0.4.26.jar (org.jetbrains.intellij:plugin-repository-rest-client:0.4.26)
04:17:51
          > Could not get resource 'https://plugins.gradle.org/m2/org/jetbrains/intellij/plugin-repository-rest-client/0.4.26/plugin-repository-rest-client-0.4.26.jar'.
04:17:51
             > Could not HEAD 'https://jcenter.bintray.com/org/jetbrains/intellij/plugin-repository-rest-client/0.4.26/plugin-repository-rest-client-0.4.26.jar'. Received status code 502 from server: Bad Gateway
04:17:51
       > Could not download kotlin-stdlib-1.1.4-3.jar (org.jetbrains.kotlin:kotlin-stdlib:1.1.4-3)
04:17:51
          > Could not get resource 'https://plugins.gradle.org/m2/org/jetbrains/kotlin/kotlin-stdlib/1.1.4-3/kotlin-stdlib-1.1.4-3.jar'.
04:17:51
             > Could not HEAD 'https://jcenter.bintray.com/org/jetbrains/kotlin/kotlin-stdlib/1.1.4-3/kotlin-stdlib-1.1.4-3.jar'.
04:17:51
```

**Comment by rodm** [comment]
I have seen that article and raised the following issue [MP-3403](https://youtrack.jetbrains.com/issue/MP-3403) to have dependencies such `plugin-repository-rest-client` published somewhere else.

**Comment by rodm** [comment]
@vbedrosova I had another look at this and have re-worked the tasks that require the dependencies shown above, the changes are available in version 1.4-beta-2 of the plugin.
